src/filetypes.c

```diff
@@ -951,7 +951,7 @@ static void load_settings(guint ft_id, GKeyFile *config, GKeyFile *configh)
 	{
 		ft->lang = tm_source_file_get_named_lang(result);
 		if (ft->lang < 0)
-			geany_debug("Cannot find tag parser '%s' for custom filetype '%s'.", result, ft->name);
+			geany_debug("Cannot find tags parser '%s' for custom filetype '%s'.", result, ft->name);
```

Comment:
> > The fact that the parsers are ctags based is just a current implementation artifact

> It's also publicly exposed, since Geany can read actual CTags format tag files.

Sure, and two non-ctags formats.  Like I said tags if a file format, thats file, the files are .tags.